corehq/apps/es/client.py

```diff
@@ -439,8 +439,19 @@ def reindex(
             "conflicts": "proceed"
         }
 
-        # Should be removed after ES 5-6 migration
-        if copy_doc_ids:
+    # Should be removed after ES 5-6 migration
+        if copy_doc_ids and source == const.HQ_USERS_INDEX_NAME:
```

Comment:
The generic `ElasticManageAdapter.reindex` method feels like the wrong place to put migration code for a specific index. However, I don't have a suggestion for a better way.

Reply:
Yeah. This is not nice but I guess it's temporary and should be deleted once we are on ES 6.